premium/usage.go

```diff
@@ -264,6 +295,31 @@ func (u *BatchUpdater) Close() error {
 	return <-u.closeError
 }
 
+func (u *BatchUpdater) getTableUsage() (usage []cqapi.UsageIncreaseTablesInner, total uint32) {
+	u.mutex.Lock()
+	defer u.mutex.Unlock()
+
+	for key, value := range u.tables {
+		usage = append(usage, cqapi.UsageIncreaseTablesInner{
+			Name: key,
+			Rows: int(value),
+		})
+	}
+
+	return usage, u.rows
+}
+
+func (u *BatchUpdater) chunkTableUsage(usage []cqapi.UsageIncreaseTablesInner, total uint32) {
```

Comment:
what does `chunk` mean in this context?

Reply:
We store total rows and table breakdowns on the usage client, so this function basically **chunks** the table usages and total rows that were sent to the API from the ones tracked on the usage client.

Does it make sense?

Reply:
Hmm, I've not heard of `chunk` being used this way; to me `chunk` means to divide something into parts, and this doesn't really seem to apply here. Maybe `subtractTableUsage`?

Reply:
Done